### PR TITLE
Homework/homework 3

### PR DESCRIPTION
**Pull request by VictorKozhevnikov**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (ubcent) [inline .vscode/launch.json]: You should add it to .gitignore

File: types/through2/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Type definitions for through2 v 2.0
```
Comment (ubcent) [inline types/through2/index.d.ts:1]: Why it here? You should install it with `npm i @types/throudh2`

**Review by ubcent — APPROVED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (VictorKozhevnikov) [reply .vscode/launch.json]: Done

File: types/through2/index.d.ts (hunk 2 above)
Comment (VictorKozhevnikov) [reply types/through2/index.d.ts:1]: I did install `@types/through2`, but I had to change it. So you see the changed file.
The TransformCallback type was outside the namespace. I moved it inside so I can import and use it.

Comment (ubcent) [reply types/through2/index.d.ts:1]: Ok. I see

**Pull request merged**